doc/xcodebuild.txt

```diff
@@ -1,31 +1,139 @@
-*xcodebuild.txt* Plugin to build and test Xcode projects from vim
+*xcodebuild.txt*
+
+                            xcodebuild.vim
+                Build and test Xcode projects from Vim
 
 ==============================================================================
-CONTENTS                               *xcodebuild-contents*
+CONTENTS                                                         *XCB-Contents*
 
+      1. About.......................... |XCB-About|
```

Comment:
should I rename the global sections too? `xcodebuild-xcodebuild_xcpretty_testing_flags` is pretty verbose.